save-orchestrator-common/src/main/kotlin/com/saveourtool/save/orchestrator/kubernetes/KubernetesManager.kt

```diff
@@ -54,6 +54,7 @@ class KubernetesManager(
             }
             spec = JobSpec().apply {
                 parallelism = replicas
+                ttlSecondsAfterFinished = TTL_AFTER_COMPLETED
```

Comment:
can we move it to `com.saveourtool.save.orchestrator.config.ConfigProperties.KubernetesSettings` ?
```
        val ttlAfterFinished: Duration = 180.seconds,
```
```suggestion
                ttlSecondsAfterFinished = kubernetesSettings.ttlAfterFinished.toInt(DurationUnit.SECONDS)
```